src/pages/product-list-page/product-list-item/product-list-item.js

```diff
@@ -43,7 +43,7 @@ const ProductListItem = ({ product }) => {
     );
     return availableSizes && availableSizes[0] ? (
       <>
-        {t('common.from') + availableSizes[availableSizes.length - 1]?.price[currency].value}
```

Comment:
I guess, we should leave this field not to confuze customers, cuz there could be different prices for different sizes => more than 1 price for product
this question should be discussed with Product Owner and Designers on the meeting

Reply:
fixed